src/content/architecture/components/lighthouse/_index.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Lighthouse"
+date: 2020-03-02T21:25:35+01:00
+weight: 10
+---
+
+Lighthouse provides DNS discovery to Kubernetes clusters connected by [Submariner](https://github.com/submariner-io/submariner) in multi-cluster connected environments. The solution is compatible with any CNI (Container Network Interfaces) plugin.
+
+## Architecture
+The below digram shows the basic Lighthouse architecture.
+
+![Lighthouse Architecture](/images/lighthouse/architecture.png)
+
+### Lighthouse Controller
+This is the central discovery controller that gathers the information from the clusters, decides what information is to be shared and distributes the information as newly defined CRDs (Kubernetes custom resources).
+
+The Lighthouse controller uses the kubefed directory of clusters through the [Admiral](https://github.com/submariner-io/admiral) library. It connects to each cluster and listens for Service CRUD. Once it is notified that a new Service is created and needs to be distributed, it creates a MultiClusterService CR which is distributed to all the clusters that are part of the control plane. The flow for update and delete will be similar.
+
+#### WorkFlow
+The typical workflow is as follows.
+
+- Lighthouse controller registers with Admiral into the kubefed list of clusters for cluster join and unjoin events.
+- Once it is notified about a join event, it retrieves the credentials from Admiral and registers for a call back on Service creation with a shared informer.
+- When a new Service is created in a cluster, the Lighthouse controller will be notified about it and it will create the Lighthouse CRD with the Service info.
+- The controller with the help of Admiral distributes the CRD to all the clusters that have joined.
```

Comment:
I think we can remove this line and other references to Admiral - I don't think talking about Admiral adds any value - it's really an implementation detail.